client/lib/plans/constants.js

```diff
@@ -148,9 +154,10 @@ export const plansList = {
 			FEATURE_NO_ADS,
 			FEATURE_WORDADS_INSTANT,
 			FEATURE_VIDEO_UPLOADS,
+			isEnabled( 'manage/advanced-seo' ) && FEATURE_ADVANCED_SEO,
```

Comment:
This is fine for this particular case but if you want to change features list/descriptions/titles etc in an abtest, use `applyTestFiltersToPlansList` fn please: https://github.com/Automattic/wp-calypso/blob/master/client/lib/plans/index.js#L206 :).